src/hexkit/providers/akafka/provider/eventsub.py

```diff
@@ -427,7 +203,7 @@ def __init__(
     def _get_event_label(event: ConsumerEvent) -> str:
         """Get a label that identifies an event."""
         return (
-            f"{event.topic} - {event.partition} - {event.offset} "
+            f"{event.topic} - {event.partition} - {event.key} - {event.offset} "
```

Comment:
```suggestion
            f"{event.topic} - {event.partition} - {event.key} - {event.offset}"
```

since the blank is already added in the next line